front/pages/w/[wId]/vaults/[vaultId]/index.tsx

```diff
@@ -29,11 +30,12 @@ export const getServerSideProps = withDefaultUserAuthRequirements<
     auth,
     context.query.vaultId as string
   );
-  if (!vault) {
+  if (!vault || (vault.canRead(auth))) {
```

Comment:
Same here, you throw not found if can read 🙈

Reply:
🙈